src/SlackConnector/Connections/Sockets/Messages/Inbound/InboundMessage.cs

```diff
@@ -9,17 +9,6 @@ internal class InboundMessage
         [JsonConverter(typeof(EnumConverter))]
         public MessageType MessageType { get; set; }
 
-        [JsonProperty("subtype")]
-        [JsonConverter(typeof(EnumConverter))]
-        public MessageSubType MessageSubType { get; set; }
-
```

Comment:
But then these attributes must be repeated on all message types as they are not inherited through the interface.